crates/router/src/utils/connector_onboarding.rs

```
@@ -34,3 +39,98 @@ pub fn is_enabled(
         _ => None,
     }
 }
+
+pub async fn check_if_connector_exists(
+    state: &AppState,
+    connector_id: &str,
+    merchant_id: &str,
+) -> RouterResult<()> {
+    let key_store = state
+        .store
+        .get_merchant_key_store_by_merchant_id(
+            merchant_id,
+            &state.store.get_master_key().to_vec().into(),
+        )
+        .await
+        .change_context(ApiErrorResponse::InternalServerError)?;
+
+    let _connector = state
+        .store
+        .find_by_merchant_connector_account_merchant_id_merchant_connector_id(
+            merchant_id,
+            connector_id,
+            &key_store,
+        )
+        .await
+        .to_not_found_response(ApiErrorResponse::MerchantConnectorAccountNotFound {
+            id: connector_id.to_string(),
+        })?;
+
+    Ok(())
+}
+
+pub async fn set_tracking_id_in_configs(
+    state: &AppState,
+    connector_id: &str,
+    connector: enums::Connector,
+) -> RouterResult<()> {
+    let timestamp = common_utils::date_time::now_unix_timestamp().to_string();
+    if state
+        .store
+        .find_config_by_key(&build_key(connector_id, connector))
+        .await
+        .is_ok()
```

do not ignore errors, atleast log it